test/e2e/apps/statefulset.go

```diff
@@ -871,6 +874,88 @@ var _ = SIGDescribe("StatefulSet", func() {
 		})
 	})
 
+	/*
+		Release: v1.19
+		Testname: StatefulSet rollover from stuck rollout
+		Description: When StatefulSet is updated with invalid image or gets similarly failing
+		on the first updated pod (highest ordinal) it should be able to update when the podSpec is corrected.
+	*/
+	framework.ConformanceIt("should rollover from stuck rollout", func() {
```

Comment:
We cannot create conformance tests directory without promotion from normal e2e tests.
Here needs to be `ginkgo.It` instead of `framework.ConformanceIt`